### PR TITLE
sw-7008 updating packagers #minor

### PR DESCRIPTION
**Pull request by Nicholassully**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Major dependencies
 
--   [Go](https://golang.org/) (>= 1.16)
+-   [Go](https://golang.org/) (>= 1.19)
```
Comment (denialanderror) [inline README.md:5]: Should this be 1.21?

File: web/template/layout/page.gotmpl [hunk 2]
```diff
@@ -65,6 +65,7 @@
                     sirius-pahub
                 {{ end }}
             {{ end }}
+            govuk-frontend-supported
```
Comment (denialanderror) [inline web/template/layout/page.gotmpl:68]: Is this meant to be here?

File: yarn-error.log
Comment (denialanderror) [inline yarn-error.log:1]: I think this should be ignored

**Review by denialanderror — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Nicholassully) [reply README.md]: aye indeed changed

File: web/template/layout/page.gotmpl (hunk 2 above)
Comment (Nicholassully) [reply web/template/layout/page.gotmpl:68]: This was the tag that we need to make sure the JS worked

File: yarn-error.log
Comment (Nicholassully) [reply yarn-error.log:1]: Removed

**Review by denialanderror — APPROVED** [review]

**Pull request merged**